frontend/dockerfile/instructions/parse_test.go

```diff
@@ -170,7 +170,7 @@ func TestErrorCases(t *testing.T) {
 		},
 		{
 			name:          "MAINTAINER unknown flag",
-			dockerfile:    "MAINTAINER --boo joe@example.com",
+			dockerfile:    "MAINTAINER --boo=ba joe@example.com",
```

Comment:
Shouldn't we be checking that we handle unknown flags correctly whether or not they have an argument? IOW a new test case for the with argument case seems like a good idea.

Reply:
Sure